python/src/ir.cc

```diff
@@ -490,6 +490,16 @@ void init_triton_ir(py::module &&m) {
            [](ModuleOp &self, FuncOp &funcOp) -> void {
              self.push_back(funcOp);
            })
+      .def("get_first_func_name",
+           [](ModuleOp &self) -> std::string {
+             std::string str;
```

Comment:
It's confusing to me that the only purpose of `str` is to be returned as empty string. Can we just create an empty string in-place on the line that returns empty string?